sktree/tree/_classes.py

```diff
@@ -1106,51 +1075,71 @@ class PatchObliqueDecisionTreeClassifier(DecisionTreeClassifier):
         ``help(sklearn.tree._tree.Tree)`` for
         attributes of Tree object.
 
-    min_patch_dims_ : array-like
-        The minimum dimensions of a patch.
-
-    max_patch_dims_ : array-like
-        The maximum dimensions of a patch.
+    feature_combinations_ : float
+        The number of feature combinations on average taken to fit the tree.
 
-    data_dims_ : array-like
-        The presumed dimensions of the un-vectorized feature vector.
+    See Also
+    --------
+    DecisionTreeClassifier : An axis-aligned decision tree classifier.
 
     Notes
     -----
-    Patches can be 2D masks that are applied onto the data matrix. Following sklearn
-    API standards, ``X`` is always a ``(n_samples, n_features)`` array even if
-    X is comprised of images, or multivariate-time series. The ``data_width`` and
-    ``data_height`` parameters are used to inform the ``PatchObliqueDecisionTreeClassifier``
-    of the original structure of the data. It is required that
-    ``data_width * data_height = n_features``.
-
-    When users pass in ``X`` to :meth:`fit`, tt is presumed that all vectorization operations
-    are done C-contiguously (i.e. the last axis is contiguous).
-
-    Note that for a patch height and width of size 1, the tree is exactly the same as the
-    decision tree, albeit with less efficiency optimizations. Therefore, it is always
-    recommended to set the range of patch heights and widths based on the structure of your
-    expected input data.
+    Compared to ``DecisionTreeClassifier``, oblique trees can sample
+    more features then ``n_features``, where ``n_features`` is the number
+    of columns in ``X``. This is controlled via the ``max_features``
+    parameter. In fact, sampling more times results in better
+    trees with the caveat that there is an increased computation. It is
+    always recommended to sample more if one is willing to spend the
+    computational resources.
+    The default values for the parameters controlling the size of the trees
+    (e.g. ``max_depth``, ``min_samples_leaf``, etc.) lead to fully grown and
+    unpruned trees which can potentially be very large on some data sets. To
+    reduce memory consumption, the complexity and size of the trees should be
+    controlled by setting those parameter values.
+    The :meth:`predict` method operates using the :func:`numpy.argmax`
+    function on the outputs of :meth:`predict_proba`. This means that in
```

Comment:
There is no `predict_proba` method inside regressors. Overall, I would go through the documentation again to make sure it matches what semantically the class/function is trying to do.